lib/src/spy.dart

```diff
@@ -0,0 +1,12 @@
+import 'dart:mirrors';
```

Comment:
Let's add a comment that this is separated so that we can keep mirrors separated.